slick.grid.js

```diff
@@ -1642,60 +1685,71 @@ if (typeof Slick === "undefined") {
    		return impactedColumns;
     }
 
+    function handleResizeableHandleDoubleClick(evt) {
+      const triggeredByColumn = evt.target.parentElement.id.replace(uid, "");
+      trigger(self.onColumnsResizeDblClick, { triggeredByColumn: triggeredByColumn.getReturnValue() });
+    }
+
     function setupColumnResize() {
       if (typeof Slick.Resizable === "undefined") {
         throw new Error('Slick.Resizable is undefined, make sure to import "slick.interactions.js"');
       }
 
-      var $col, j, k, c, pageX, columnElements, minPageX, maxPageX, firstResizable, lastResizable;
+      var j, k, c, pageX, minPageX, maxPageX, firstResizable, lastResizable;
       var frozenLeftColMaxWidth = 0;
-      columnElements = $headers.children();
-      columnElements.find(".slick-resizable-handle").remove();
-      columnElements.each(function (i, e) {
-        if (i >= columns.length) { return; }
+
+      const children = getHeaderChildren();
+      for (let i = 0; i < children.length; i++) {
+        const child = children[i];
+        const handles = child.querySelectorAll(".slick-resizable-handle");
+        handles.forEach(function (handle) {
+          handle.remove();
+        });
+
+        if (i >= columns.length) 
+          return;
+
         if (columns[i].resizable) {
           if (firstResizable === undefined) {
             firstResizable = i;
           }
           lastResizable = i;
         }
-      });
-      if (firstResizable === undefined) {
-        return;
       }
 
-      columnElements.each(function (i, colElm) {
+      if (firstResizable === undefined)
+        return;
+
+      for (let i = 0; i < children.length; i++) {
+        const colElm = children[i];
+
         if (i >= columns.length) { return; }
         if (i < firstResizable || (options.forceFitColumns && i >= lastResizable)) {
           return;
```

Comment:
since this `return` was previously used within a jQuery `.each()` method, this `return` will not work inside a `querySelectorAll` loop because it will stop the loop too early. Basically we need to change the `return` to a `continue` for this to work the same as a jQuery each loop